inc/class-wpseo-utils.php

```diff
@@ -1098,6 +1098,33 @@ public static function is_woocommerce_active() {
 		return class_exists( 'Woocommerce' );
 	}
 
+	/**
+	 * Determines whether the plugin is active for the entire network.
+	 *
+	 * @return bool Whether or not the plugin is network-active.
+	 */
+	public static function is_plugin_network_active() {
+		static $network_active = null;
+
+		if ( ! is_multisite() ) {
+			return false;
+		}
+
+		// Store the result in a variable to prevent multiple lookups.
+		if ( $network_active === null ) {
+			$network_active_plugins = wp_get_active_network_plugins();
+
+			if ( strpos( wp_normalize_path( WPSEO_FILE ), wp_normalize_path( WPMU_PLUGIN_DIR ) ) === 0 || in_array( WP_PLUGIN_DIR . '/' . WPSEO_BASENAME, $network_active_plugins, true ) ) {
+				$network_active = true;
+			}
+			else {
+				$network_active = false;
```

Comment:
Couldn't this be the default value of the variable and only override it if the above if-statement is true? This would make the `else` unnecessary.